Platforms/QemuQ35Pkg/ConfigDataGfx/ConfigDataGfx.inf

```diff
@@ -27,20 +27,16 @@
   BaseLib
   DebugLib
   MemoryAllocationLib
-  ConfigBlobBaseLib
   PeimEntryPoint
   PeiServicesLib
   PrintLib
+  ConfigKnobShimLib
 
 [Guids]
-  gSetupConfigPolicyVariableGuid                # CONSUMES
   gPolicyDataGFXGuid                            # CONSUMES
 
 [Ppis]
-  gEfiPeiReadOnlyVariable2PpiGuid   ## CONSUMES
   gPeiPolicyPpiGuid                 ## CONSUMES
 
 [Depex]
-  gEfiPeiReadOnlyVariable2PpiGuid AND
   gPeiPolicyPpiGuid
-

```

Comment:
This is odd, locally it shows me there is a newline at the end of the file. I'll go through and fix the no newlines at the end.